content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -158,81 +158,50 @@ If you use VS Code and see the error `An import path cannot end with a '.ts' ext
-      Deno Deploy requires a GitHub repository and you create that in [Create a repository and push to GitHub](#8-create-a-repository-and-push-to-github).
-
-      ![Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs](../../../doc-images/data-platform/projects/create-01-paste-connection-string.png)
-
-1. Create a Data Proxy connection string.
-
-   1. On the **Get started** screen, click **Create a new connection string**.
-   1. Enter a name for the connection string and click **Create**.
-   1. Copy the `prisma://` connection string to your clipboard.
-
-      ![Prisma Data Platform - Create project - Paste database connection string, select Data Proxy region, enable static IPs](../../../doc-images/data-platform/projects/create-03-copy-data-proxy-connection-string.png)
-
-   1. Save the connection string in a password manager or another safe location.
-   1. Click **Skip and continue to Data Platform** to open the Data Browser for your project.<br /><br />
-      When you test your application later, you come back to the Data Browser to review the records that your application creates.
-
-### What's next
-
-Next in this guide, you use the connection string in your local `.env` file to connect to your database through the Data Proxy.
-
-## 6. Configure the Data Proxy connection string in your environment
+Setup an Accelerate connection string and proceed to the next step.
 
-With the Data Proxy connection string copied, you can replace the direct PostgreSQL connection string that you used to create the database schema in your `.env` file.
+## 6. Configure Prisma Accelerate in your environment
 
-Prisma Client for the Data Proxy does not read `.env` files by default, so you must also install `dotenv-cli` locally.
+With the Accelerate connection string copied, you can replace the direct PostgreSQL connection string that you used to create the database schema in your `.env` file.
 
 ### Steps
 
-1. Install `dotenv-cli`.
+1. Install the Prisma Accelerate client extension
 
-   ```terminal
-   npm install dotenv-cli
+   ```bash
+   npm install @prisma/extension-accelerate
    ```
 
-2. Add the Data Proxy connection string to the `.env` file. Also, comment out the direct connection string.
+2. Add the Accelerate connection string to the `.env` file. Also, comment out the direct connection string.
 
    ```bash file=.env
+   DATABASE_URL="DATABASE_URL="prisma://accelerate.prisma-data.net/?api_key=__API_KEY__""
+   # Previous database connection
    # DATABASE_URL="postgres://..."
-   DATABASE_URL="prisma://..."
    ```
 
+3. Add the Prisma Accelerate extension to your Prisma Client
+
+   ```typescript
+   import { PrismaClient } from './generated/client/deno/edge.ts'
+   import { withAccelerate } from '@prisma/extension-accelerate'
```

Comment:
What do I need the extension for in a pure connection pooling use case like for Deno Deploy?

Reply:
@janpio could you elaborate on this a bit further?
 You need the extension to extend PrismaClient and use it as a connection pool. Am I missing something?

Reply:
The extension is not needed for that. You can just use the `prisma://` connection string to achieve that.

Reply:
Ahhh, that makes sense, Thanks! I'll update it.